bin/rtlcss.js

```diff
@@ -13,18 +14,18 @@ let input, output, directory, ext, config, currentErrorcode, arg
 const args = process.argv.slice(2)
 let shouldBreak = false
 
-process.on('exit', function () { process.reallyExit(currentErrorcode) })
+process.on('exit', () => { process.reallyExit(currentErrorcode) })
 
-function printWarning () {
-  console.warn(chalk.yellow.apply(this, printWarning.arguments))
+function printWarning (args) {
+  console.warn(chalk.yellow(args))
 }
```

Comment:
@XhmikosR These are not equivalent, isn't it better to have them as ?
```js
function printWarning (...args) {
  console.warn(chalk.yellow.apply(this, args))
}
```

Reply:
Oops sorry, my bad.

```js
function printWarning (...args) {
  console.warn(chalk.yellow(args))
}
```

The problem is that the previous code does not work in strict mode.

Reply:
Then:

```js
function printWarning (...args) {
  console.warn(chalk.yellow(...args))
}
```

Reply:
Sorry, third time is the charm 😔 Doing too many things at the same time.